VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -212,10 +213,11 @@ public void showCourseDiscussionAddPost(@NonNull Activity activity, @Nullable Di
         activity.startActivity(addPostIntent);
     }
 
-    public void showCourseDiscussionComments(Context context, DiscussionComment comment, DiscussionThread discussionThread) {
+    public void showCourseDiscussionComments(@NonNull Context context, @NonNull DiscussionComment comment, @NonNull DiscussionThread discussionThread, boolean isNewCommentPosted) {
```

Comment:
Maybe add a new default method definition that calls to this one, with the last parameter provided as `false` by default.